downstream/modules/platform/proc-controller-project-add-permissions.adoc

```diff
@@ -0,0 +1,37 @@
+[id="proc-controller-project-add-permission"]
+
+= Adding and removing project permissions
+
+Use the folowing procedure to add permissions.
```

Comment:
folowing > following.

Should there be a semicolon after permissions, to be consistent with the opening sentence of the previous procedure?